pontoon/projects/views.py

```diff
@@ -149,7 +149,9 @@ def ajax_notifications(request, slug):
     project = get_object_or_404(
         Project.objects.visible_for(request.user).available(), slug=slug
     )
-    available_locales = project.locales.order_by("name")
+    available_locales = project.locales.prefetch_project_locale(project).order_by(
+        "name"
+    )
```

Comment:
Brings moderate performance gains by significantly replacing the number of DB calls - one for each Locale - with a Prefetch command.